doc/user-docs/UsingJsonSettings.md

```diff
@@ -69,9 +69,26 @@ For example, here's a sample of the default keybindings:
         // etc.
     ]
 }
+```
+
+A string value for `"keys"` can be used and is interpreted
```

Comment:
This is weirdly passive-voice. Consider "You can also use a single keychord string as the value of `keys`. It will be treated as a chord of length one. This will allow you to simplify the above snippet as follows:"